x-pack/plugins/security_solution/public/timelines/components/flyout/header/index.tsx

```diff
@@ -286,63 +279,26 @@ const TimelineDescriptionComponent: React.FC<FlyoutHeaderProps> = ({ timelineId
   );
   const dispatch = useDispatch();
 
-  const onReadMore: MouseEventHandler<HTMLButtonElement> = useCallback(
-    (event: MouseEvent<HTMLButtonElement>) => {
-      dispatch(
-        setActiveTabTimeline({
-          id: timelineId,
-          activeTab: TimelineTabs.notes,
-          scrollToTop: true,
-        })
-      );
-    },
-    [dispatch, timelineId]
-  );
+  const onReadMore = useCallback(() => {
```

Comment:
Lol, just found that this existed. Never had a timeline description long enough 😅